src/mongo/db/fts/fts_query_impl.cpp

```diff
@@ -133,8 +138,13 @@ Status FTSQueryImpl::parse(TextIndexVersion textIndexVersion) {
 
     std::unique_ptr<FTSTokenizer> tokenizer(ftsLanguage.getValue()->createTokenizer());
 
-    _addTerms(tokenizer.get(), positiveTermSentence, false);
-    _addTerms(tokenizer.get(), negativeTermSentence, true);
+	if(isNgram){
+		_addTermsForNgram(tokenizer.get(), positiveTermSentence, false);
+		_addTermsForNgram(tokenizer.get(), negativeTermSentence, true);
+	}else{
+		_addTerms(tokenizer.get(), positiveTermSentence, false);
+		_addTerms(tokenizer.get(), negativeTermSentence, true);
+	}
```

Comment:
This block of code seems to use tabs for indentation. Please change it to 4 spaces.